### PR TITLE
Add english doc about pause and resume method (reference #31).

### PR DESCRIPTION
**Pull request by osaris**

### CODE REVIEW

File: Documentation/En/Index.xyl [hunk 1]
```diff
@@ -111,6 +111,9 @@ echo $bench;
  *     __global__  ||||||||||||||||||||||||||||||||||||||||||||||||||||    45ms, 100.0%
  *     aMark       |||||||||||||                                           11ms,  24.8%
  */</code></pre>
+  <p>We can also pause all marks and get what marks was running with
```
Comment (Hywan) [inline Documentation/En/Index.xyl]:
```diff
- was
+ were
```

File: Documentation/En/Index.xyl [hunk 2]
```diff
@@ -111,6 +111,9 @@ echo $bench;
...
+  <code>Hoa\Bench\Bench::pause</code> method. The
+  <code>Hoa\Bench\Bench::resume</code> method allow to restart a set of marks.</p>
```
Comment (Hywan) [inline Documentation/En/Index.xyl]:
```diff
- allow
+ allows
```

**Comment by Hywan** [comment]
Thank you for the PR!

File: Documentation/En/Index.xyl (hunk 1 above)
Comment (jubianchi) [inline Documentation/En/Index.xyl]:
```diff
- what
+ which
```

?

Comment (jubianchi) [inline Documentation/En/Index.xyl]:
```diff
- with
+ with the ...
```

File: Documentation/En/Index.xyl [hunk 3]
```diff
@@ -111,6 +111,9 @@ echo $bench;
  *     __global__  ||||||||||||||||||||||||||||||||||||||||||||||||||||    45ms, 100.0%
  *     aMark       |||||||||||||                                           11ms,  24.8%
  */</code></pre>
+  <p>We can also pause all marks and get what marks were running with the
```
Comment (Hywan) [inline Documentation/En/Index.xyl]: `what` vs. `which`?

Comment (osaris) [inline Documentation/En/Index.xyl]: same meaning ?

Comment (Hywan) [inline Documentation/En/Index.xyl]: I guess.

Comment (osaris) [inline Documentation/En/Index.xyl]: ```which``` is more formal imo

Comment (Hywan) [inline Documentation/En/Index.xyl]: It's up to you to decide then. I will merge this just after your decision.

Comment (jubianchi) [inline Documentation/En/Index.xyl]: `what` => `quoi`
`which` => `quel(le)`

it seems more natural to use "which" although  they seem to have the same meaning... As @Hywan you choose :p

**Comment by Hywan** [comment]
Thanks!

**Pull request merged**